SDPi_Supplement/asciidoc/volume2/mdpws-compression/tf2-ch-a-mdpws-compression.adoc

```diff
@@ -0,0 +1,56 @@
+==== MDPWS Compression Option
+
+<<ieee_11073_20702_2016>> (MDPWS) defines EXI as the designated means to compress XML data. However, EXI is lagging in adoption. Therefore, instead of using EXI compression, it is recommended to use HTTP compression with gzip (RFC 1952) as the minimum agreement between clients and servers. One issue with that is the missing official support for HTTP compression in HTTP request messages. Request headers are allowed to include Content-Encoding, but those might not be accepted by HTTP servers.
+
+.R0005
+[sdpi_requirement#r0005,sdpi_req_level=should]
+****
+HTTP clients should request gzip compression by using Accept-Encoding with "gzip".
+****
+===== Example of compression request:
```

Comment:
I like the idea of making examples collapsible - at least when being part of a requirement. Would you mind if we agree on moving the example into the requirement box, removing the title and making the examples a listing?

```
 .R0005
[sdpi_requirement#r1005,sdpi_req_level=should]
****
HTTP clients should request gzip compression by using Accept-Encoding with "gzip".

.Examples
[%collapsible]
====
.Request header to request compression
[source]
----
GET /path/to/resource HTTP/1.1

Host: www.example.com

Accept-Encoding: gzip
----
[...]
====
****
```